### PR TITLE
Benny/tez address

### PR DESCRIPTION
**Pull request by bennycio**
Changes:

- **Added** new `ChainPubKeyInput` and `ChainPubKey` types to the graphQL schema
- **Added** respective go types
- **Changed** verify signature to take in a pub key

### CODE REVIEW

File: service/user/user.go [hunk 1]
```diff
@@ -329,35 +328,36 @@ func UpdateUser(pCtx context.Context, userID persist.DBID, username string, bio
...
+// 	if pInput.WalletType != persist.WalletTypeEOA {
+// 		if auth.NewNoncePrepend+nonce != pInput.Nonce && auth.NoncePrepend+nonce != pInput.Nonce {
+// 			return auth.ErrNonceMismatch
+// 		}
+// 	}
+
+// 	sigValidBool, err := multichainProvider.VerifySignature(pCtx, pInput.Signature, nonce, chainAddress, pInput.WalletType)
+// 	if err != nil {
+// 		return err
+// 	}
```
Comment (bennycio) [inline service/user/user.go:352]: commented out because of this part, and it is not in use, we can probably just delete this

Comment (Robinnnnn) [reply service/user/user.go:352]: WEN ADMIN MERGE @radazen

Comment (Robinnnnn) [reply service/user/user.go:352]: in all seriousness, @tbezman made a good point that tezos users may accidentally create a new account that they want to combine with their ETH one. something to think about hehe

still got a few weeks until launch!

**Review by Robinnnnn — APPROVED** [review]
muy fast

**Comment by Robinnnnn** [comment]
actually @bennycio i think this needs to be updated as well: 
image

**Pull request merged**